mne/viz/_3d.py

```diff
@@ -666,8 +856,8 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
 
     for key in surfs.keys():
         surfs[key] = transform_surface_to(surfs[key], coord_frame, mri_trans)
-    src_rr = apply_trans(mri_trans, src_rr)
-    src_nn = apply_trans(mri_trans, src_nn, move=False)
+    # src_rr = apply_trans(mri_trans, src_rr)
+    # src_nn = apply_trans(mri_trans, src_nn, move=False)
```

Comment:
... oh, that example (and probably you) are reading it from the forward solution. Yes it could be converted there already. So we should check `src[0]['coord_frame']` to choose whether to use `mri_trans` or `head_trans`